src/Room.js

```diff
@@ -319,6 +320,18 @@ function notificationCallback (data) {
     return this.kuzzle.emitEvent('jwtTokenExpired');
   }
 
+  if (data.controller === 'document') {
+    data.type = 'document';
+    data.document = new Document(this.collection, data.result._id, data.result._source);
+    delete data.result;
+  }
+
+  if (data.controller === 'realtime') {
+    data.type = 'user';
+    data.user = {count: data.result.count};
```

Comment:
For the ref: here is the updated one: https://github.com/kuzzleio/sdk-javascript/pull/147#issuecomment-267339624